ci/terraform/oidc/dns.tf

```diff
@@ -89,24 +89,25 @@ resource "aws_api_gateway_domain_name" "oidc_api" {
 }
 
 data "aws_cloudformation_export" "oidc_cloudfront_distribution_id" {
-  count = var.oidc_cloudfront_dns_enabled ? 1 : 0
+  count = var.oidc_cloudfront_enabled ? 1 : 0
   name  = local.oidc_cloudfront_id_export_name
 }
 
 data "aws_cloudfront_distribution" "oidc_cloudfront_distribution" {
-  count = var.oidc_cloudfront_dns_enabled ? 1 : 0
+  count = var.oidc_cloudfront_enabled ? 1 : 0
   id    = data.aws_cloudformation_export.oidc_cloudfront_distribution_id[0].value
 }
 
 resource "aws_route53_record" "oidc_api" {
+  count   = var.oidc_cloudfront_enabled ? 1 : 0
   name    = local.oidc_api_fqdn
   type    = "A"
   zone_id = aws_route53_zone.oidc_api_zone.zone_id
 
   alias {
     evaluate_target_health = true
-    name                   = var.oidc_cloudfront_dns_enabled ? data.aws_cloudfront_distribution.oidc_cloudfront_distribution[0].domain_name : aws_api_gateway_domain_name.oidc_api.regional_domain_name
-    zone_id                = var.oidc_cloudfront_dns_enabled ? data.aws_cloudfront_distribution.oidc_cloudfront_distribution[0].hosted_zone_id : aws_api_gateway_domain_name.oidc_api.regional_zone_id
+    name                   = data.aws_cloudfront_distribution.oidc_cloudfront_distribution[0].domain_name
+    zone_id                = data.aws_cloudfront_distribution.oidc_cloudfront_distribution[0].hosted_zone_id
   }
```

Comment:
Should the new flag not go here instead?

Reply:
Yeah, I've add back the old fallback values